### PR TITLE
feat: Appsmith ai

### PR DESCRIPTION
**Pull request by harshilp24**
## Checklist
I have:
- [ ] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

**Review by jnikhila — APPROVED** [review]

**Comment by bharath31** [comment]
@harshilp24 could you add a screenshot with an actual query and response? nit: giving the query a custom name would look better.

File: website/docs/connect-data/reference/appsmith-ai.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+The Generate Text action allows you to generate text based on specific prompts.
+
+
+<dd>
+
+#### Prompt
+
+
+
+This field allows you to input prompts for text generation. You can also add data from queries or JSObjects using mustache bindings `{{}}`.
```
Comment (bharath31) [inline website/docs/connect-data/reference/appsmith-ai.md:34]: Maybe add an example for a binding along with a hyperlink to the documentation page that talks about binding?

File: website/docs/connect-data/reference/appsmith-ai.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Labels refer to predefined categories or classes assigned to the analyzed text. These labels are essential for organizing and identifying the nature of the text content. 
+
+For example, if you are classifying product reviews, labels could include categories such as "Positive," "Negative," or "Neutral."
+
+</dd>
+
+<dd>
+
+#### Additional Instructions
```
Comment (bharath31) [inline website/docs/connect-data/reference/appsmith-ai.md:80]: Let users know that they can add descriptions for labels in the additional instructions. 

eg: 
Labels: Urgent, High priority, Low

Additional Instructions: Use the following labels descriptions: 
Urgent: Issues that prevent a customer from using the product
High priority: Issues that are nasty but still makes the product usable
Low priority: Issues that can be lived with

File: website/docs/connect-data/reference/appsmith-ai.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+
+
+This property allows you to provide additional instructions to fine-tune the text summarization process. You can use this field to offer specific guidance or adjustments to the AI.
```
Comment (bharath31) [inline website/docs/connect-data/reference/appsmith-ai.md:113]: Add an example and let users know that they can control the summary length

**Review by jnikhila — APPROVED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**